helper/src/commonMain/kotlin/com/algolia/instantsearch/helper/filter/range/FilterRangeConnection.kt

```diff
@@ -21,4 +21,34 @@ public fun <T> FilterRangeConnector<T>.connectView(
     view: NumberRangeView<T>
 ): Connection where T : Number, T : Comparable<T> {
     return viewModel.connectView(view)
-}
\ No newline at end of file
+}
+
+public inline fun <reified T> FilterRangeConnector<T>.connectSearcher(
+    searcherSingleIndex: SearcherSingleIndex,
+    attribute: Attribute
+): Connection where T : Number, T : Comparable<T> {
+    return viewModel.connectSearcher(searcherSingleIndex, attribute)
+}
+
+public inline fun <reified T> FilterRangeConnector<T>.connectSearcher(
+    searcherSingleIndex: SearcherSingleIndex,
+    attribute: Attribute,
+    mapper: (Number) -> T
+): Connection where T : Number, T : Comparable<T> {
+    return viewModel.connectSearcher(searcherSingleIndex, attribute, mapper)
+}
+
+public inline fun <reified T> FilterRangeViewModel<T>.connectSearcher(
+    searcherSingleIndex: SearcherSingleIndex,
+    attribute: Attribute
+): Connection where T : Number, T : Comparable<T> {
+    return FilterRangeConnectionSearcher(this, searcherSingleIndex, attribute, T::class)
+}
+
+public inline fun <reified T> FilterRangeViewModel<T>.connectSearcher(
+    searcherSingleIndex: SearcherSingleIndex,
+    attribute: Attribute,
+    mapper: (Number) -> T
```

Comment:
`mapper` is not being used in this function

Reply:
Yes, fixed :D thanks